mne/io/reference.py

```diff
@@ -561,3 +561,79 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
         inst.drop_channels(drop_channels)
 
     return inst
+
+@verbose
+def set_laplace_reference(inst, central_ch, adjacent_chs, copy=True, 
```

Comment:
can you add this to an example?

I'm a bit wary of adding functions that are not accompanied by examples. It's how we realized that `set_bipolar_reference` was broken since it was too slow to be usable ...